docs/tasks/administer-cluster/pvc-protection.md

```diff
@@ -1,144 +0,0 @@
----
-reviewers:
-- msau42
-- jsafrane
-title: Persistent Volume Claim Protection
```

Comment:
Should we just revise this task doc instead of removing it?

Reply:
The PR https://github.com/kubernetes/website/pull/7620 renamed the `docs/tasks/administer-cluster/pvc-protection.md` file to `docs/tasks/administer-cluster/storage-object-in-use-protection.md` and forgot to remove the `docs/tasks/administer-cluster/pvc-protection.md` file that is not used anywhere in the docs. So removal of the `docs/tasks/administer-cluster/pvc-protection.md` file is a fix of the PR https://github.com/kubernetes/website/pull/7620
IMHO, I should have created a separate PR for the fix (`docs/tasks/administer-cluster/pvc-protection.md` file removal) and not mix it with this PR.

Reply:
Okay. Thanks for the clarification.